src/applications/gi/utils/render.jsx

```diff
@@ -27,16 +28,16 @@ export const renderSchoolClosingAlert = result => {
           />
         );
       }
+      return (
```

Comment:
@shawkey-daniel @hughes-dustin From my understanding It looks like we are not getting the correct message, moving this up would give us the `A campus at this school will be closing soon` message when `(currentDate > schoolClosingDate) is false`. 

The one question I have is would `schoolClosingOn` ever be null when we expect to display `A campus at this school will be closing soon`